source/usage-examples/deleteMany.txt

```diff
@@ -15,29 +20,6 @@ Delete Many Documents
 You can delete more than one document using the ``DeleteMany()`` synchronous
 method or the ``DeleteManyAsync()`` asynchronous method on a collection object.
 
-Example
--------
```

Comment:
I: I think you should consider leaving this heading in all of the usage examples, since removing it makes it so that the tree navigation on some of the pages starts with "Expected Result" like [here](https://preview-mongodbmongokart.gatsbyjs.io/csharp/docsp-27681-move-sample-classes/usage-examples/insertOne/) and [here](https://preview-mongodbmongokart.gatsbyjs.io/csharp/docsp-27681-move-sample-classes/usage-examples/updateMany/)

Reply:
Good thought